vertica/datadog_checks/vertica/data/conf.yaml.example

```diff
@@ -0,0 +1,164 @@
+init_config:
+  ## @param global_custom_queries - list - optional
+  ## See `custom_queries` defined below.
+  ##
+  ## Global custom queries can be applied to all instances using the
+  ## `use_global_custom_queries` setting at the instance level.
+  #
+  # global_custom_queries:
+  #   - query: <QUERY>
+  #     columns: <COLUMNS>
+  #     tags: <TAGS>
+
+instances:
+
+    ## @param db - string - required
+    ## The name of the database to establish a connection with.
+    #
+  - db: <DATABASE_NAME>
+
+    ## @param server - string - required
+    ## The hostname used to connect to the database.
+    #
+    server: <SERVER>
```

Comment:
Should we put localhost as a placeholder like we do for MySQL: https://github.com/DataDog/integrations-core/blob/master/mysql/datadog_checks/mysql/data/conf.yaml.example#L10 ?

Reply:
I don't think so